core/rio/nquads/src/main/java/org/eclipse/rdf4j/rio/nquads/NQuadsParser.java

```diff
@@ -35,143 +26,82 @@
 
 	protected Resource context;
 
-	@Override
-	public RDFFormat getRDFFormat() {
-		return RDFFormat.NQUADS;
-	}
-
-	@Override
-	public synchronized void parse(final InputStream inputStream, final String baseURI)
-			throws IOException, RDFParseException, RDFHandlerException {
-		if (inputStream == null) {
-			throw new IllegalArgumentException("Input stream can not be 'null'");
-		}
+	/*--------------*
+	 * Constructors *
+	 *--------------*/
```

Comment:
Fwiw we don't really use these kinds of markers anymore as they do more harm than good. Not a big deal, you can leave it in here, but no need to add this in for any future contributions :)

Reply:
I don't like these markers either, I just noticed they are present for a number of parsers/writers and wanted to be consistent. But I will remove them :)